bigquery_etl/view/__init__.py

```diff
@@ -194,7 +194,10 @@ def publish(self, target_project=None, dry_run=False):
 
         # avoid checking references since Jenkins might throw an exception:
         # https://github.com/mozilla/bigquery-etl/issues/2246
-        if self._valid_view_naming():
+        if (
+            any(str(self.path).endswith(p) for p in SKIP_VALIDATION)
+            or self._valid_view_naming()
```

Comment:
I think we need the opposite filter here:
```suggestion
            not any(str(self.path).endswith(p) for p in SKIP_VALIDATION)
            and self._valid_view_naming()
```

Reply:
There are some views, like `telemetry.client_probe_counts_v1`, that we want to publish although their validation would fail (view name != file path). So I it should be correct as it is.